src/server/qgscapabilitiescache.h

```diff
@@ -59,6 +59,10 @@ class SERVER_EXPORT QgsCapabilitiesCache : public QObject
      */
     void removeCapabilitiesDocument( const QString &path );
 
+  public slots:
+    //! Removes changed entry from this cache
```

Comment:
documentation

Reply:
and maybe call directly removeCapabilitiesDocument, and get rid of dbg message

Reply:
done :+1: